org.eclipse.jdt.ls.filesystem/src/org/eclipse/jdt/ls/core/internal/filesystem/JLSFsUtils.java

```diff
@@ -110,6 +114,27 @@ static boolean isProjectMetadataFile(IPath location) {
         return true;
     }
 
+    /**
+     * Check whether the given location is excluded
+     * @param location file location.
+     * @return whether the given location is excluded.
+     */
+    public static boolean isExcluded(IPath path) {
+        if (path != null && JDTLSFilesystemActivator.getResourcePatterns() != null) {
+            for (String segment: path.segments()) {
+                for (Pattern pattern: JDTLSFilesystemActivator.getResourcePatterns()) {
```

Comment:
Same here. Space between the element and the `:`.

Reply:
Fixed